package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/cookie-policy-banner",
-  "version": "1.1.7",
+  "version": "1.1.8",
```

Comment:
Unrelated to this change, but this got pulled into my `package-lock.json` because the `1.1.8` version bump was only reflected in the `package.json` file.